omero/sysadmins/whatsnew.txt

```diff
@@ -1,33 +1,4 @@
-What's new for OMERO 5.2 for sysadmins
+What's new for OMERO 5.3 for sysadmins
 ======================================
 
```

Comment:
* Ice 3.6 will be the default so we may want to keep it
* web app will be ``pip installable``